src/json.rs

```diff
@@ -0,0 +1,143 @@
+use serde::de::{Deserialize, DeserializeOwned};
```

Comment:
There should be module-level documentation (`//!` doc comments) explaining the purpose of this module.